docs/reference/style-guides/supplemental/software-architecture-implementation.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Software Architecture Implementation
```

Comment:
Should this be more specific? If I'm looking at the title list in `adrs/` and I see this, I'd have to click into it to understand what this was covering specifically.